setup.py

```diff
@@ -0,0 +1,38 @@
+""" Setup.py for packaging log-anomaly-detector as library """
+import os
+from setuptools import setup
+
+os.system('pip install git+https://github.com/sevamoo/SOMPY.git' +
+          '@76b60ebd6ffd550b0f7faaf632451dfd68827bf7')
+os.system('pip install ' +
+          'git+https://github.com/jaybaird/python-bloomfilter.git' +
+          '@2bbe01ad49965bf759e31781e6820408068862ac')
```

Comment:
Calling pip while pip installs packages seems to be not a good approach. What about directly adding these requirements to `REQURED_PKG` below?

Reply:
I'm going to open a git issue for that. Thank you @fridex